### DISCUSSION TITLE
Attribution for "Sponsors"

### DISCUSSION THREAD

**Discussion by nilsnolde** [valhalla/valhalla#3142] [General]
I got the question from a client if his company would be somehow visibly attributed for funding feature implementations (in this case GTFS implementation). I think that's a fair request, so I wanted to open the discussion to see what you guys are thinking.

From what I've seen, attribution of sponsors often happens on the homepage or so, not necessarily on Github. But there's no community homepage for valhalla (as e.g. OSRM or QGIS). What would be a good place you think? I guess a markdown that's not toooo hidden? CHANGELOG, README, something else? Or not at all? 

What's Mapbox's opinion, doesn't say anywhere that most contributors are from Mapbox. Was that just never thought about or intentionally not done for some reason?

**Comment by kevinkreiser** [comment]
The project is officially a Linux Foundation project in which the project is run by the voting committee/board. The committee is comprised of @dgearhart @dnesbitt61 @gknisely @kdiluca @kevinkreiser.  We should update the readme to reflect that.

I personally think it would be nice to write a little blurb about the major contributors over the years as its own section in the readme: Mapzen, Mapillary, Mapbox, Navitia, Gis Ops, many others..

**Comment by kevinkreiser** [comment]
> Was that just never thought about or intentionally not done for some reason?

No reason that I know of! At the very beginning we had in the license file Mapzen and then Mapillary when they worked with us on map matching. After the project went into the linux foundation we updated the license to just be "valhalla contributors" as it was easier to deal with.

**Comment by nilsnolde** [comment]
> I personally think it would be nice to write a little blurb about the major contributors over the years as its own section in the readme: Mapzen, Mapillary, Mapbox, Navitia, Gis Ops, many others..

Yep, my feeling too. In our case, we (sometimes at least) get paid to implement features, how to deal with those companies? IMHO it'd be enough to basically show logos, size of those more or less reflecting the size of the contribution? Or split it up to maintainer group and contributor group?

Anyone else having an opinion here?

**Reply by mandeepsandhu** [reply]
As you noted, since there's no "project page" other than this repo, it might be good to add sponsorship or contributor details on the top-level README. Just My 2¢
